postgres/tests/unit.tftest.hcl

```diff
@@ -224,6 +224,11 @@ run "aws_db_instance_unit_test" {
     error_message = "Should be: true"
   }
 
+  assert {
+    condition     = aws_db_instance.default.deletion_protection == true
+    error_message = "Should be: true"
+  }
```

Comment:
We should probably have a test that verifies it can be set to false. I suspect this is true of other things with defaults too.

Reply:
Fair point. I'll look into it